### PR TITLE
Bitsight/adds guard against no status code

### PR DESCRIPTION
**Pull request by sethborden**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
Updates the BitSight test module to better handle cases where a 200 response isn't received from BitSight servers.

## Screenshots
N/A

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by tallieber** [comment]
Hey @sethborden, what was happen when the status code wasn't 200, what was the behaviour?  Where this code acts differently with status code that is not 200? (I DM you also on slack)

File: Packs/BitSight/Integrations/BitSightForSecurityPerformanceManagement/BitSightForSecurityPerformanceManagement.py [hunk 1]
```diff
@@ -125,14 +125,15 @@ def test_module(client):
     Anything else will fail the test.
     """
     res = client.get_companies_guid()
-    res_status_code = res.get("status_code", None)
-    if res_status_code == 200:
+
+    available_guids = {c["guid"] for c in res["companies"]}
+    requested_guid = demisto.params()["guid"]
```
Comment (tallieber) [inline Packs/BitSight/Integrations/BitSightForSecurityPerformanceManagement/BitSightForSecurityPerformanceManagement.py:130]: Better use .get()

Comment (tallieber) [reply Packs/BitSight/Integrations/BitSightForSecurityPerformanceManagement/BitSightForSecurityPerformanceManagement.py:130]:
```suggestion
    available_guids = {c.get("guid", "") for c in res["companies"]}
    requested_guid = demisto.params().get("guid", "")
```

**Review by tallieber — CHANGES_REQUESTED** [review]

**Comment by sethborden** [comment]
@tallieber here's the video of the the non-working version of the code along with the working version of the code. In order to update the code I cloned the existing version of the BitSight code into a BitSight Development integration, but didn't commit those changes.  These changes should update the "production" version of the code.

https://user-images.githubusercontent.com/3753337/135869828-5b935a60-006f-490e-a6b2-db146ff2e804.mov

**Comment by tallieber** [comment]
Hey @sethborden thank you for the video!

In order to fix the branching issue, you need to pull and merge master into your working branch, we cannot accept changes into the master branch.

**Pull request merged**